src/pages/AdvancedSearch.vue

```diff
@@ -187,11 +187,20 @@ export default class AdvancedSearchPage extends Vue {
       });
     }
   }
-  private onFormChange(input: { name: string; value: string }) {
+  private async onFormChange(input: { name: string; value: string }) {
     const { name, value } = input;
     let processedName = name;
     let processedVal = inputProcessor(name, value);
 
+    // Fallback
+    if (!processedVal && ["vote", "generatorPublicKey"].includes(name)) {
```

Comment:
One thing with the fallback in this location, is that `inputProcessor` has a `return publicKey ? publicKey : value;` so it will always return a value and never trigger this code? It would be better to have it inside the `lookupPublicKey` method and use the fallback in case there was no successful lookup.